src/app/components/modals/LoadingModalService.js

```diff
@@ -18,8 +19,8 @@ class LoadingModalService {
   open() {
     if (!this._modalInstance) {
       this._modalInstance = this._uibModal.open({
-        template: '<span us-spinner></span>',
-        windowTemplate: '<div uib-modal-transclude></div>',
+        template: '<md-progress-circular md-mode="indeterminate"></md-progress-circular>',
+	windowTemplate: '<div uib-modal-transclude></div>',
```

Comment:
can we keep the formatting consistent. I'll add a program to "auto" format, but for now lets just to it manually

Reply:
I'm not sure if you're using vscode or not. If so, you can install the "prettier" plugin and set it to auto format when a file is saved so you don't have to think about it.